website/src/utils/networkUtils.ts

```diff
@@ -0,0 +1,139 @@
+// Networks that should use the "mono" icon variant
+export const MONO_ICON_NETWORKS = [
+  'vana',
+  'vana-moksha',
+  'xlayer-mainnet',
+  'xlayer-sepolia',
+  'zksync-era',
+  'zksync-era-sepolia',
+  'sonic',
+  'soneium-testnet',
+  'soneium',
+  'linea-sepolia',
+  'linea',
+  'lens-testnet',
+  'expchain-testnet',
+  'autonomys-taurus',
+  'arweave-mainnet',
+  'fraxtal',
+  'lumia',
+  'mbase',
+]
```

Comment:
Nice! Ideally this is not needed in the future, but in the meantime, this is a neat solution.